### PR TITLE
Fixes #9879: Update datasource status when a data source data are updated

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/9879

### CODE REVIEW

File: rudder-web/src/main/scala/com/normation/rudder/web/rest/datasource/DataSourceApiService.scala [hunk 1]
```diff
@@ -58,7 +58,7 @@ class DataSourceApiService(
   def getSources() : Box[JValue] = {
     for {
       sources <- dataSourceRepo.getAll
-      data = sources.values.map(restDataSerializer.serializeDataSource(_))
+      data = sources.values.map(s => restDataSerializer.serializeDataSource(s._1))
```
Comment (fanf) [inline rudder-web/src/main/scala/com/normation/rudder/web/rest/datasource/DataSourceApiService.scala]: Status may be huge. I believe we need a "getAllDataSources"  without the status part here (and corresponding implementation of course)

**Comment by fanf** [comment]
Cool!

File: rudder-web/src/main/scala/com/normation/rudder/web/rest/datasource/DataSourceApiService.scala (hunk 1 above)
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/rest/datasource/DataSourceApiService.scala]: I'm ok with you making an update

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by peckpeck** [comment]
This code should go to the plugin repo.
It may serve as a reference in a future implementation.

**Pull request closed without merging**